.github/workflows/main.yml

```diff
@@ -0,0 +1,102 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  PYTEST_ADDOPTS: "--color=yes"
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9, pypy3]
+        django-version: [">=2.2,<2.3", ">=3.0,<3.1", ">=3.1,<3.2"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          echo "Python ${{ matrix.python-version }} -> Django ${{ matrix.django-version }}"
+          python -m pip install "Django${{ matrix.django-version }}"
+          python -m pip install -q -e .
+          python -m pip install pytest pytest-cov pytest-django
+          echo "Django: `django-admin --version`"
+          python --version
+      - name: Run tests
+        run: pytest --cov elevate --cov-report term-missing --cov-fail-under 100
```

Comment:
Upload to Coveralls appears to be missing

Reply:
It's not clear to me how to upload to Coveralls from Pytest, which I alluded to in the PR description. Any suggested methods for doing that?

Reply:
https://github.com/marketplace/actions/coveralls-github-action should do the trick

Reply:
Yeah, I tried that already but got an error I couldn't quite figure out how to solve: https://github.com/justinmayer/django-elevate/runs/1341105989?check_suite_focus=true#step:2:9

Reply:
You've made coverage a separate job, rather than an extra step in the "Test" job. Just add "Update Coveralls" after "Run tests" and you should be good.

Reply:
After wasting an inordinate amount of time, I still couldn't get Coveralls to upload reports via the above GitHub Action, so I replaced Coveralls with Codecov.